mmgen/models/gans/base_translation_model.py

```diff
@@ -0,0 +1,135 @@
+from abc import ABCMeta, abstractmethod
+from copy import deepcopy
+
+import torch.nn as nn
+
+from ..builder import MODELS
+
+
+@MODELS.register_module()
+class BaseTranslationModel(nn.Module, metaclass=ABCMeta):
```

Comment:
This design seems nice to me. However, this file should under a separate folder, such as `mmgen/models/translation_models/base_translation_model.py`.
(Maybe github has bugs on name and path display after using `git mv`.)